features/sct-section-header.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Header stuff
+  Scenario: Basic
+    Given a section
+     Then section.header is a Header object
+
+    Given a header having a definition
+     Then header.is_linked_to_previous is False
+
+    Given a header having a definition
+     Then header.text is the text contained in the header
```

Comment:
* The "Feature:" description needs to be more descriptive. There's a pattern to it, you can look at the other feature files to see a bunch of examples.
* There's vertical spacing between the Feature description and the Scenarios (2 blank lines). Check out the others for examples. Basically each section in the .feature files is separated by 2 blank lines.
* Each scenario (Given: block) has its own title. It should also be descriptive. Check out some of the other feature files.
* Scenarios are separated by two blank lines
* I've renamed this file `sct-access-header.feature` This will eventually hold the .first_page_header, .footer, etc. scenarios when we get to them.
* We should put the other two in a separate file named `hdr-props.feature`. A general rule of thumb is scenarios that start with 'Given an x ...' should go in a file that starts with the three-letter abbreviation for x. But lets leave them out for now and stick to this first one.
* Acceptance tests that don't pass yet need the `@wip` decorator. This allows them to be skipped during the full test run while their implementation is work in progress. A flag on the command line tells behave whether to skip them or not, something like '--tags=-wip'. You run them when you're looking for the next error and skip them when regression testing before merge.